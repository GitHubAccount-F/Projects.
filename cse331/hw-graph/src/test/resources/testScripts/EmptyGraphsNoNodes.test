#Tests to see if empty graphs contain no nodes

#Creates two empty graphs
CreateGraph graph1
CreateGraph graph2

#Lists out the nodes inside the two graphs
ListNodes graph1
ListNodes graph2