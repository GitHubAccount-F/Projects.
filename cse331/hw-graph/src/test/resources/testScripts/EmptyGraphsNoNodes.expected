#Tests to see if empty graphs contain no nodes

#Creates two empty graphs
created graph graph1
created graph graph2

#Lists out the nodes inside the two graphs
graph1 contains:
graph2 contains: